core/docker/Dockerfile

```diff
@@ -11,16 +11,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
The refactoring to the docker file should be done in a separate commit (e.g., in case we have to revert just that if there are issues)